api/envoy/extensions/transport_sockets/tls/v3/tls_spiffe_validator_config.proto

```diff
@@ -55,6 +80,21 @@ message SPIFFECertValidatorConfig {
     config.core.v3.DataSource trust_bundle = 2;
   }
 
+  enum TrustBundleFormat {
+    TRUST_BUNDLE_MAP = 0;
+  }
+
+  message TrustBundlesSource {
+    TrustBundleFormat format = 1;
+    config.core.v3.DataSource source = 2;
+  }
```

Comment:
It's a little more complex then I think. I actually only want to rename the original field to `trust_bundles`. And in case new format is coming, we can add the enum at that time in the `SPIFFECertValidatorConfig` directly  and needn't to add the enum now. And also needn't a wrapper.

By that way, we don't pay any additional complexity if actually no any new format emerges (I suggest to change the name just because naming change is simple and free. I actually agree that no more format is the most cases.)